### PR TITLE
Initialize odoo model (some fix)

### PR DESCRIPTION
**Pull request by AndriiK139MCAF**

### CODE REVIEW

File: mo_hr_timesheets/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
```
Comment (MORPSmcaf) [inline mo_hr_timesheets/__manifest__.py:1]: remove comments

File: mo_hr_timesheets/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+{
+    'name': "mo_hr_timesheets",
```
Comment (MORPSmcaf) [inline mo_hr_timesheets/__manifest__.py:3]: rename as "Custom HR  timesheet"

File: mo_hr_timesheets/__manifest__.py [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+    'summary': "Timshet somthing somthing",
```
Comment (MORPSmcaf) [inline mo_hr_timesheets/__manifest__.py:4]: Custom HR timesheet and Employee Information

File: mo_hr_timesheets/__manifest__.py [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+    'version': '16.0.1.0.0',
+    'website': "https://mcaf.nb.ca/en/",
+    'author': "MCAF",
+    'category': "Appointments",
```
Comment (MORPSmcaf) [inline mo_hr_timesheets/__manifest__.py:8]: 'category': "HR"

File: mo_hr_timesheets/__manifest__.py (hunk 1 above)
Comment (AndriiK139MCAF) [reply mo_hr_timesheets/__manifest__.py:1]: fix

File: mo_hr_timesheets/__manifest__.py (hunk 2 above)
Comment (AndriiK139MCAF) [reply mo_hr_timesheets/__manifest__.py:3]: fix

File: mo_hr_timesheets/__manifest__.py (hunk 3 above)
Comment (AndriiK139MCAF) [reply mo_hr_timesheets/__manifest__.py:4]: fix

File: mo_hr_timesheets/__init__.py [hunk 5]
```diff
@@ -0,0 +1,3 @@
+# -*- coding: utf-8 -*-
```
Comment (MORPSmcaf) [inline mo_hr_timesheets/__init__.py:1]: remove comments

File: mo_hr_timesheets/models/__init__.py [hunk 6]
```diff
@@ -0,0 +1,3 @@
+# -*- coding: utf-8 -*-
```
Comment (MORPSmcaf) [inline mo_hr_timesheets/models/__init__.py:1]: remove comments

File: mo_hr_timesheets/__init__.py (hunk 5 above)
Comment (AndriiK139MCAF) [reply mo_hr_timesheets/__init__.py:1]: fix

File: mo_hr_timesheets/models/__init__.py (hunk 6 above)
Comment (AndriiK139MCAF) [reply mo_hr_timesheets/models/__init__.py:1]: fix

**Review by MORPSmcaf — APPROVED** [review]

**Pull request merged**